### PR TITLE
blog: stryker-js 7

### PR DESCRIPTION
**Pull request by nicojs**
TODO:

- [x] vitest
- [x] tap
- [x] allowEmpty
- [x] cleanTempDir always
- [x] breaking changes

### CODE REVIEW

**Comment by nicojs** [comment]
@danny12321 @odinvanderlinden  would you mind taking a look?

File: blog/2023-06-05-announcing-stryker-js-7.0.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    "testFiles": ["test/**/*.spec.js"]
+  }
+}
+```
+
+See the [tap runner documentation](/docs/stryker-js/tap-runner) for more details.
+
+💐 Special thanks go to [@danny12321](https://github.com/danny12321) for implementing this plugin, and [isaacs](https://github.com/isaacs) for help in the design ❤
+
+## 🪹 Allow empty
```
Comment (danny12321) [inline blog/2023-06-05-announcing-stryker-js-7.0.md]: This emoji is not recognized by my browser, which is chrome.

**Review by danny12321 — CHANGES_REQUESTED** [review]
The text looks good, but left one comment about an emoji.

**Review by danny12321 — CHANGES_REQUESTED** [review]
The text looks good, but left one comment about an emoji.

**Comment by nicojs** [comment]
Thanks! I've changed the empty nest (🪹) emoji with an empty mailbox (📭) emoji :)

**Pull request merged**